### ISSUE TITLE
client does not sync any file or directory permissions

### ISSUE THREAD

**Issue by tessus** [owncloud/client#4885]
### Expected behaviour
Client should sync permissions of files and directories as well.

### Actual behaviour
Creates all files/directories with the umask of the user under which the client runs.

### Comments
This might be the intended behavior, because Windows does not use standard file permissions, but in that case the client could just ignore Windows. I can't believe that all other clients have to lack this functionality because of Windows.

### Steps to reproduce
1. copy a file with permission `-rw-------` to a sync folder
2. wait on another machine until the file shows up
3. check the permissions of that file -> `-rw-r--r--`
 
### Server configuration
Operating system: Linux 64bit

Web server: Apache 2.4.20

Database: Mysql 5.6.25

PHP version: 5.6.21

ownCloud version: 8.2.4

Storage backend: file system

### Client configuration
Client version: Version 2.2.0 (build 3358)

Operating system: OSX 10.11.5 and 10.9.5

OS language: us-en

Installation path of client: /Applications/owncloud.app

### Logs

1. Output of `owncloud --logwindow` or `owncloud --logfile log.txt`

2. Web server error log: no errors

3. ownCloud log (data/owncloud.log): no errors

**Comment by RealRancor** [comment]
Looks to me like a duplicate of https://github.com/owncloud/client/issues/3199

**Comment by tessus** [comment]
@RealRancor not so sure about that. you will have to decide, if it is a duplicate. in the other ticket, the OP talks about execute permissions and permissions created via the UI. I didn't even know that you could change the permissions via the web interface.

My ticket is about `client <-> server <-> client` flow. The other ticket is about `server <-> client`.

Also, the other ticket has been open for a year now, even though the first comment states the following _adding to milestone as it's currently in the top-10_.
It looks to me as if it was postponed every time it popped up on the radar again. Maybe you should close the other ticket and leave this one open.

**Comment by RealRancor** [comment]
@tessus I don't have any permissions and possibilities to decide something :-)

**Comment by ogoffart** [comment]
Duplicate of https://github.com/owncloud/core/issues/6983

**Comment by tessus** [comment]
@ogoffart yes, I agree that is in a duplicate, but the other ticket has been open for **2 and a half years**!

Unless you move it out of the backlog stage, it will NEVER be implemented. Maybe the developers will understand, if a lot of tickets are open regarding this topic, that it might be rather important for people.

So what's happening here is the following: All related tickets are and will be closed, because it's a duplicate. The original ticket on the other hand is and will be ignored. But maybe I'm wrong. I will check back in another 2 years, by which time I will tell you: I told you so.
